protocols/indexer/test/Indexer-unit.js

```diff
@@ -462,4 +462,31 @@ contract('Indexer Unit Tests', async accounts => {
       equal(intents.length, 1, 'intents array should be size 1')
     })
   })
+
+  describe('Test getScore', async () => {
+    it('should fail if the index does not exist', async () => {
+      await reverted(
+        indexer.setIntent(tokenOne, tokenTwo, 1000, aliceLocator, {
+          from: aliceAddress,
+        }),
+        'INDEX_DOES_NOT_EXIST'
+      )
+    })
+
+    it('should retrieve the score on a token pair for a user', async () => {
+      // create index
+      await indexer.createIndex(tokenOne, tokenTwo, {
+        from: aliceAddress,
+      })
+
+      let stakeAmount = 1000
+      // set an intent staking 0
```

Comment:
I don't understand this comment. Isn't it staking 1000?

Reply:
looks like a copy paste error